libcontainer/container_linux.go

```diff
@@ -2321,6 +2321,18 @@ func (c *Container) bootstrapData(cloneFlags uintptr, nsMaps map[configs.Namespa
 		})
 	}
 
+	// write boottime and monotonic time ns offsets.
+	if c.config.Namespaces.Contains(configs.NEWTIME) && c.config.TimeOffsets != nil {
```

Comment:
I suspect we want this check to be done in the validator (at is stands, the code will silently ignore `timensOffsets` being configured without `CLONE_NEWTIME` enabled). But I can do this in a separate PR when I add the necessary integration tests for this feature.